openpype/hosts/flame/api/lib.py

```diff
@@ -272,3 +291,46 @@ def rescan_hooks():
         flame.execute_shortcut('Rescan Python Hooks')
     except Exception:
         pass
+
+
+def get_metadata(project_name, _log=None):
+    from adsk.libwiretapPythonClientAPI import (
+        WireTapClient,
+        WireTapServerHandle,
+        WireTapNodeHandle,
+        WireTapStr
+    )
+
+    class GetProjectColorPolicy(object):
```

Comment:
Why this is a class? It's used only inside scope of `get_metadata` function and the object is thrown away at the moment of `get_metadata` function end. It would make more sense to have it as function (combination of `__init__` and `process`).

Reply:
yes you are right. this code has been coppied from other source and it is here because I needed to quickly test something. But since I believe it will be used in future, I am keeping it here. It is not used anywhere now and in future it will be rewritten.